repos/public-user/src/Module/GetConfig.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace PublicUser\Module;
+
+use Zend\Config\Factory;
+
+trait GetConfig
+{
+    private $config;
+
+    public function getConfig()
+    {
+        if ($this->config) {
+            return $this->config;
+        }
+        // Load our composer dependencies.
+        $this->loadVendor();
+        $this->config = Factory::fromFiles(
+            glob(__DIR__ . '/../../config/*.config.*')
+        );
+
+        return $this->config;
+    }
+
+    public function loadVendor()
+    {
+        if (file_exists(__DIR__ . '/build/vendor-dist/autoload.php')) {
```

Comment:
Why do we still need to do this dance? We don't deploy modules without composer anymore.

Reply:
Will remove from all in separate PR (this was in due to splitting out of `Module.php`) and make sure it still works without

Reply:
Ill try removing these and tidying them in separate PR, make sure that it will work correctly. I think we are using composer exclusively now.